src/shims/unix/fd.rs

```diff
@@ -187,33 +198,133 @@ impl FileDescription for NullOutput {
         Ok(Ok(bytes.len()))
     }
 }
+/// Structure contains both the file description and its unique identifier.
+#[derive(Clone, Debug)]
+pub struct FileDescWithID<T: FileDescription + ?Sized> {
```

Comment:
Should be `...WithId`, using the normal Rust capitalization rules

Reply:
Same with `FdID`, should be `FdId`